com/zaehlwerk.go

```diff
@@ -17,7 +18,7 @@ type Zaehlwerk struct {
 	Einheit           mengeneinheit.Mengeneinheit     `json:"einheit,omitempty" validate:"required"`                                   // Die Einheit der gemessenen Größe
 	Zaehlerstaende    Zaehlerstaende                  `json:"zaehlerstaende,omitempty"`                                                // A list of Zaehlerstand (Non BO4E Standard)
 	Verwendungszwecke []Verwendungszweck              `json:"verwendungszwecke,omitempty"`                                             // Verwendungungszweck der Werte Marktlokation
-
+	Verbrauchsart     []verbrauchsart.Verbrauchsart   `json:"verbrauchsart,omitempty"`                                                 // Stromverbrauchsart/Verbrauchsart Marktlokation
```

Comment:
is it actually a slice? https://github.com/Hochfrequenz/BO4E-dotnet/blob/b6dc240a3bd45f6b65d2d00a4a2a68f01909174f/BO4E/COM/Zaehlwerk.cs#L101